supabase/migrations/20230811105133_log_entries.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE log_entries (
+  id SERIAL PRIMARY KEY,
+  repo_name TEXT,
+  org_name TEXT,
+  comment_id INT,
+  issue_number INT,
+  log_message TEXT,
+  level INT,
```

Comment:
Why is the data type of `level` INT? Log level should be "debug", "info", "warn", "error"... so I prefer making it string.

Reply:
I recommended that because it's better for database index. We can convert to string in the UI